nodewatcher/modules/analysis/rogue_nodes/tests.py

```diff
@@ -0,0 +1,28 @@
+import json
+import io
+import os
+
+from django import test as django_test
+
+from nodewatcher.modules.analysis.rogue_nodes.tasks import rogue_node_detection_algorithm
+
+
+class RogueDetectionTest(django_test.TestCase):
+
+    def test_rogue_detection_algorithm(self):
+        print('testing rogue_node_detection_algorithm')
```

Comment:
What about the print statements that show exactly which test case failed? Should I remove those as well?